dask_cuda/utils.py

```diff
@@ -764,3 +765,73 @@ def get_rmm_memory_resource_stack(mr) -> list:
         if isinstance(mr, rmm.mr.StatisticsResourceAdaptor):
             return mr.allocation_counts["current_bytes"]
     return None
+
+
+def enable_rmm_memory_for_library(lib_name: str) -> None:
```

Comment:
Can you please move `enable_rmm_memory_for_library`/`_setup_rmm_for_torch`/`_setup_rmm_for_cupy` to `plugins.py`?

Reply:
Resolved by: https://github.com/rapidsai/dask-cuda/commit/33871acd1ace384e1237f2caaf4a041a2c1b7cbf